package.json

```diff
@@ -8,11 +8,11 @@
   ],
   "main": "lib/index.js",
   "scripts": {
-    "clean": "del lib/ coverage/ example/build example/dist",
+    "clean": "rm -rf lib/ coverage/ example/build example/dist",
```

Comment:
If you're a windows-user, master contains an update which should make this command work again